include/bluetooth/mesh/sensor.h

```diff
@@ -64,18 +68,94 @@ enum bt_mesh_sensor_sampling {
 	BT_MESH_SENSOR_SAMPLING_COUNT,
 };
 
+/** Delta threshold type. */
+enum bt_mesh_sensor_delta {
```

Comment:
I realized that the new API doesn't need a public `enum bt_mesh_sensor_delta` at all (since it's not included in the cadence struct anymore). Will move it into `sensor.c` when the old API is disabled (and give it a better name).